src/DebugEngineHost/HostEvaluateRequestContext.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DebugEngineHost
+{
+    /// <summary>
+    /// Provides acceess for MIEngine to determine if it is in an EvaluateRequest with specific contexes that do not
+    /// exist in enum_PARSEFLAGS.
+    /// </summary>
+    public sealed class HostEvaluateRequestContext
+    {
+        /// <summary>
+        /// Returns the boolean if the current EvaluareRequest's context is a clipboard context.
+        /// </summary>
+        /// <returns>true if it is in an evaluate request with a clipboard context</returns>
+        public static bool IsClipboardContext()
+        {
+            throw new NotImplementedException();
```

Comment:
Don't want VS to throw an exception during an evaluate request, should return `false`.